students/alexLaws/lesson04/mailroom.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+donor_list = []
+donor_list.append({'Name': 'Ryan Moore', 'Donation': 500})
+donor_list.append({'Name': 'Ryan Moore', 'Donation': 250})
+donor_list.append({'Name': 'Ted Laws', 'Donation': 1000})
+donor_list.append({'Name': 'Ted Laws', 'Donation': 100})
+donor_list.append({'Name': 'Ben Snell', 'Donation': 150})
+donor_list.append({'Name': 'Andrew Crawford', 'Donation': 2000})
+donor_list.append({'Name': 'Andrew Crawford', 'Donation': 2000})
+donor_list.append({'Name': 'Andrew Crawford', 'Donation': 4000})
+donor_list.append({'Name': 'Beth Ross', 'Donation': 400})
+
+
+def user_action():
+    print("Choose an action from this list:")
+    print("Send a Thank You")
+    print("Create a Report")
+    print("Quit")
+    while True:
+        option = input("What is your choice: ")
+        if option.lower() in ["send a thank you", "create a report", "quit"]:
+            return option
+            break
+        else:
+            print("That is not an option. Choose again.")
+
+
+def dnrs(x):
```

Comment:
plz don't use x for var names 🙂 you can use short hand var names for comprehension loops where it is throw away and clear what you are doing.

this tells me no indication what `x` is and i had to trace down the code to find out

Reply:
function would probably be better called `get_donor_names` ?